### PR TITLE
Translate zh tw v0.0.1

### PR DESCRIPTION
**Pull request by JuenTingShie**
### Check if this PR fulfills these requirements:
* [x] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [x] I have read the [**CONTRIBUTING**](../../master/Contributing.md) document.
* [x] I made sure there wasn't another [Pull Request](../../../pulls) opened for the same update/change
* [x] I have successfully tested my changes locally

#### Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Feature change (non-breaking change which change behaviour of an existing functionality)
- [ ] Improvement (non-breaking change which improve functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Translate
### Description

I add zh_TW translate to this repo, please have a see.
and also, [**CONTRIBUTING**](../../master/Contributing.md) document is not found, maybe need to update?

### CODE REVIEW

File: resources/language/resource.language.zh_tw/strings.po [hunk 1]
```diff
@@ -0,0 +1,1092 @@
...
+msgid "Enable VP9 profiles (disable if it causes artifacts)"
+msgstr "啟用VP9（如果導致假影請禁用）"
+
+msgctxt "#30138"
+msgid "The background services may not yet be available if you just started Kodi/the add-on. In this case, try again in a moment."
+msgstr "如果您剛啟動Kodi/套件程式，則可能無法使用背景服務。在這種情況下，請稍後重試。"
+
+msgctxt "#30139"
+msgid "Enable IPC over HTTP (use when Addon Signals timeout messages appear)"
+msgstr "啟用基於HTTP的IPC（在套件程式提示信號超時資訊時使用）"
```
Comment (CastagnaIT) [inline resources/language/resource.language.zh_tw/strings.po:548]: `Addon Signals` must not be translated

File: resources/language/resource.language.zh_tw/strings.po [hunk 2]
```diff
@@ -0,0 +1,1092 @@
...
+msgid "Another device handles the auto-updates of the shared library"
+msgstr "另一台設備處理共用資料庫的自動更新"
+
+msgctxt "#30212"
+msgid "There is no device that handles the auto-updates of the shared library"
+msgstr "沒有設備可以處理共用資料庫的自動更新"
+
+msgctxt "#30213"
+msgid "InputStream Helper settings..."
+msgstr "InputStream助手設置..."
```
Comment (CastagnaIT) [inline resources/language/resource.language.zh_tw/strings.po:838]: `InputStream Helper` must not be translated

File: resources/language/resource.language.zh_tw/strings.po [hunk 3]
```diff
@@ -0,0 +1,1092 @@
...
+msgid "Choose the login method[CR](if the login with E-Mail/Password always returns \"incorrect password\", then use Authentication key, instructions in the GitHub ReadMe)"
+msgstr "選擇登入方式[CR]（如果使用電子郵件/密碼登入總是返回“不正確的密碼”，請使用身份驗證金鑰，查看GitHub讀我檔案中的說明）"
+
+msgctxt "#30341"
+msgid "E-Mail/Password"
+msgstr "電子信箱/密碼"
+
+msgctxt "#30342"
+msgid "Authentication key"
+msgstr "身份驗證金鑰"
```
Comment (CastagnaIT) [inline resources/language/resource.language.zh_tw/strings.po:976]: I think that this is better keep the english version,
the `Authentication key` is referred to the software/script name that generate this file
or do you think users understand anyway?

File: resources/language/resource.language.zh_tw/strings.po [hunk 4]
```diff
@@ -0,0 +1,1092 @@
...
+
+msgctxt "#30343"
+msgid "The selected file is not compatible or corrupted"
+msgstr "所選文件不相容或已損壞"
+
+msgctxt "#30344"
+msgid "The authentication key file is expired"
```
Comment (CastagnaIT) [inline resources/language/resource.language.zh_tw/strings.po:983]: same of previous comment, let me know

**Review by CastagnaIT — CHANGES_REQUESTED** [review]
Hi thanks for the new translation!
only minor thing to fix/check

File: resources/language/resource.language.zh_tw/strings.po (hunk 3 above)
Comment (JuenTingShie) [reply resources/language/resource.language.zh_tw/strings.po:976]: OK, maybe keep "Authentication key" is more easily to find for user and not letting translate misconstrue

**Review by CastagnaIT — APPROVED** [review]

**Pull request merged**